setup.py

```diff
@@ -97,6 +97,13 @@
         "Programming Language :: Python :: 3",
         "Topic :: Scientific/Engineering :: Physics"
     ],
+    url="https://github.com/cherab",
+    project_urls=dict(
+        Tracker="https://github.com/cherab/core/issues"
+        Documentation="https://cherab.github.io/documentation/"
+    ),
+    long_description="file: README.md",
```

Comment:
This syntax is lifted from setup.cfg, which is declarative rather than programmatic. I think for setup.py you need to read the contents of the file into a string and pass that as long description:
```python
with open("README.md") as f:
    long_description = f.read()
```